gef-ui/server/controllers/unissued-facilities/validation.js

```diff
@@ -220,7 +222,22 @@ const facilityValidation = async (body, query, params) => {
   }
 
   if (coverStartDateIsFullyComplete) {
-    coverStartDate = set(new Date(), { year: coverStartDateYear, month: coverStartDateMonth - 1, date: coverStartDateDay });
+    let value;
```

Comment:
you can assign a default value to `value`. and override it inside the `if` statement. in this scenario you wouldn't need the `else` part